packages/mask/shared-ui/locales/en-US.json

```diff
@@ -877,12 +877,25 @@
     "plugin_pets_dialog_btn": "Confirm",
     "plugin_pets_dialog_btn_share": "Share",
     "plugin_pets_dialog_preview": "Preview",
+    "plugin_pets_dialog_unverified": " (unverified)",
     "plugin_pets_dialog_created": "Created by NonFFriend",
     "plugin_pets_dialog_powered": "Powered by RSS3",
     "powered_by": "Powered by",
     "plugin_pets_dialog_success": "Your Non-Fungible Friend has been set up successfully",
     "plugin_pets_dialog_fail": "Setting failed, please try later",
     "plugin_pets_dialog_check_title": "Show NFT friends on the profile page.",
+    "plugin_pets_dialog_menu_change": "Change",
+    "plugin_pets_dialog_menu_ski": "Ski",
+    "plugin_pets_dialog_menu_about": "About us",
+    "plugin_game_name": "GAME",
+    "plugin_game_list_title": "GameList",
+    "plugin_game_list_rank": "Rank",
+    "plugin_game_list_play": "Play",
+    "plugin_game_list_play_error": "Please connect wallet",
+    "plugin_game_list_play_evm_error": "Please connect to EVM wallet",
+    "plugin_game_share_btn": "Share",
+    "plugin_game_share_title": "Share",
+    "plugin_game_dialog_info": "Share this game to twitter",
```

Comment:
Could you move them into plugin's own locale files? You can do it along this PR or another one.

And here is a guide for you. https://github.com/DimensionDev/Maskbook/wiki/Draft:-Plugin-scope-i18n-guide